File: customlayout/src/main/java/com/zxj/customlayout/MainActivity.java
package com.zxj.customlayout;

import android.content.Intent;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.btn_view).setOnClickListener(v -> {
            startActivity(new Intent(this, LayoutViewActivity.class));
        });

        findViewById(R.id.btn_group).setOnClickListener(v -> {
            startActivity(new Intent(this, LayoutGroupActivity.class));
        });
        startActivity(new Intent(this, LayoutGroupActivity.class));
    }
}

File: touch/src/main/java/com/zxj/touch/MainActivity.kt
package com.zxj.touch

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.zxj.touch.drag.DragActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        startActivity(Intent(this, DragActivity::class.java))
        finish()
    }
}